e2e/relayer/relayer.go

```diff
@@ -15,8 +15,10 @@ const (
 	Rly    = "rly"
 	Hermes = "hermes"
 
-	cosmosRelayerRepository = "damiannolan/rly" //"ghcr.io/cosmos/relayer"
-	cosmosRelayerUser       = "100:1000"        // docker run -it --rm --entrypoint echo ghcr.io/cosmos/relayer "$(id -u):$(id -g)"
+	hermesRelayerRepository = "ghcr.io/informalsystems/hermes"
+	hermesRelayerUser       = "1000:1000"
+	rlyRelayerRepository    = "damiannolan/rly" //"ghcr.io/cosmos/relayer"
```

Comment:
should this be changed back?

Reply:
nope, these are used when creating the relayer in `newHermesRelayer`:

https://github.com/cosmos/ibc-go/blob/77bc38dfe837867a56ebcb78db21acdfb03b82af/e2e/relayer/relayer.go#L61

Reply:
sorry I mean't the `damiannolan/rly` string?

Reply:
ah, right, that's pending on https://github.com/cosmos/relayer/issues/1102